Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -373,7 +373,7 @@ void BluetoothReal::WaitForHCICommandComplete(const u16 opcode)
   for (int tries = 0; tries < 100; ++tries)
   {
     if (libusb_interrupt_transfer(m_handle, HCI_EVENT, buffer.data(),
-                                  static_cast<int>(buffer.size()), &actual_length, 20) == 0 &&
+                                  static_cast<int>(buffer.size()), &actual_length, TIMEOUT) == 0 &&
```

Comment:
I mean, I'm confused.
I'm trying to understand. If games expect a 20ms timeout max, why this method call can take up to 2 seconds to execute? And why the 200ms timeout everywhere else? The max. latency "bottleneck" for Bluetooth adapters seems to be in this 20ms interrupt transfer, and it seems like the code is set up so that it can take up to 2 seconds to do the HCI command complete verification anyway.
I'm not even sure of what happens when this fail, as it seems to me that there just needs to be a pause to check for the completion of the command, but that it doesn't check if the completion really happened (or if it didn't happen after trying for 2 seconds and the method just returned)
Keep in mind that I'm only trying to understand, and that I don't code in C++ so I might be totally wrong!